sound/soc/sof/ipc.c

```diff
@@ -216,13 +216,15 @@ static int tx_wait_done(struct snd_sof_ipc *ipc, struct snd_sof_ipc_msg *msg,
 	} else {
 		/* copy the data returned from DSP */
 		ret = msg->reply_error;
-		if (msg->reply_size)
```

Comment:
@juimonen looks good only a minor request to move the comment above about copying the data to where it is actually moved to now

Reply:
ok